mozdef_util/mozdef_util/elasticsearch_client.py

```diff
@@ -160,6 +166,9 @@ def save_event(self, body, index='events', doc_id=None, bulk=False):
         event.add_required_fields()
         return self.__save_document(index=index, body=event, doc_id=doc_id, bulk=bulk)
 
+    def save_dashboard(self, body, dash_name, doc_id=None):
+        return self.es_connection.index(index='.kibana', doc_type='dashboard', body=body, id=doc_id)
```

Comment:
doc_type is "doc" not "dashboard" in es6.

Reply:
This will get removed once https://github.com/mozilla/MozDef/pull/1248 is merged.